opencsp/common/lib/geometry/Vxyz.py

```diff
@@ -494,3 +497,19 @@ def origin(cls):
     #     for x, y in zip(xs, ys):
     #         zs.append(func(x, y))
     #     return cls([xs, ys, zs])
+
+    def draw_list(
+        self,
+        figure: rcfr.RenderControlFigureRecord | v3d.View3d,
+        close: bool = None,
+        style: rcps.RenderControlPointSeq = None,
+        label: str = None,
+    ) -> None:
+        """Calls figure.draw_xyz_list(self.data.T) with the default arguments in place for any None's."""
+        kwargs = dict()
+        for key, val in [('close', close), ('style', style), ('label', label)]:
+            if val is not None:
+                kwargs[key] = val
+
+        view = figure if isinstance(figure, v3d.View3d) else figure.view
+        view.draw_xyz_list(self.data.T, **kwargs)

```

Comment:
Why is this called "draw_list" when there are no lists involved?

Reply:
I'm basing it off of the method draw_xyz_list() in View3d.py. But you're right. I changed the name for this class (but not in View3d).